Description: for now, disable some features to avoid NEW trip

Index: curl/Cargo.toml
===================================================================
--- curl.orig/Cargo.toml
+++ curl/Cargo.toml
@@ -74,15 +74,7 @@ force-system-lib-on-osx = ["curl-sys/for
 http2 = ["curl-sys/http2"]
-mesalink = ["curl-sys/mesalink"]
-ntlm = ["curl-sys/ntlm"]
-poll_7_68_0 = ["curl-sys/poll_7_68_0"]
-protocol-ftp = ["curl-sys/protocol-ftp"]
-rustls = ["curl-sys/rustls"]
-spnego = ["curl-sys/spnego"]
 ssl = ["openssl-sys", "openssl-probe", "curl-sys/ssl"]
 static-curl = ["curl-sys/static-curl"]
 static-ssl = ["curl-sys/static-ssl"]
-upkeep_7_62_0 = ["curl-sys/upkeep_7_62_0"]
-zlib-ng-compat = ["curl-sys/zlib-ng-compat", "static-curl"]
 [target."cfg(all(unix, not(target_os = \"macos\")))".dependencies.openssl-probe]
 version = "0.1.2"
 optional = true
